shellclear/src/sensitive-patterns.yaml

```diff
@@ -1,11 +1,11 @@
-- name: AWS Env Access Key
-  test: AWS_ACCESS_KEY_ID=([0-9a-zA-Z/+]{0,100})
-  secret_group: 1
-  id: aws_end_access_key
-- name: AWS Env Secret Key
-  test: AWS_SECRET_ACCESS_KEY=([0-9a-zA-Z/+]{0,100})
-  secret_group: 1
-  id: aws_env_secret_key
+#- name: AWS Env Access Key
```

Comment:
it look we have a lot of patterns that checks patterns only by key. i think we need to solve it as part of this PR (maybe in a new PR and merge it to this PR)